solana-programs/anchor/audius-data/lib/constants.ts

```diff
@@ -1,5 +1,29 @@
-export const DUMMY_CN_WALLET_ADDRESSES = [
-    "0x25A3Acd4758Ab107ea0Bd739382B8130cD1F204d",
-    "0x71B55d7bDe40D751087A27e2072F0fF8cacA400a",
-    "0xb4bD6911d3F633A1F7B14D955E68061F6f845027"
-]
\ No newline at end of file
+// Generated with --deterministic flag from ganache cli container.
```

Comment:
some day we should share this as a global config in the protocol (or mono) repo. awesome.